test_conformance/computeinfo/main.cpp

```diff
@@ -655,7 +655,7 @@ void dumpConfigInfo(cl_device_id device, config_info* info)
             log_info("\t%s == %zu\n", info->opcode_name, info->config.sizet);
             break;
         case type_cl_ulong:
-            log_info("\t%s == %lld\n", info->opcode_name, info->config.ull);
+            log_info("\t%s == %lu\n", info->opcode_name, info->config.ull);
```

Comment:
Should not be `%llu` ?

Reply:
I am getting “Format specifies type 'unsigned long long' but the argument has type 'cl_ulong' (aka 'unsigned long')” if I do that. But using `%lu` will be a problem on Win64.
Looking around, there are some standard macros to handle that: https://en.cppreference.com/w/cpp/header/cinttypes. I will switch to using those macros instead.

Reply:
This should be addressed in the updated version.

Reply:
I get hint in MSVC that ull variable has type `typedef unsigned long long cl_ulong` Then for MSVC cl_ulong is long long int

Reply:
Right, because on Win64 `unsigned long` is only 32 bits so `unsigned long long` is needed in that scenario, but on Unix `unsigned long` is already 64 bits so `cl_ulong` is just an `unsigned long` there. I had forgotten about that difference when writing the initial version.

Could you please check if the updated pull request works fine with MSCV? The macro `PRIX64` should handle both cases automatically (by changing its definition based on the platform).

Reply:
Yes I've verified looks ok. Then LGTM.

Reply:
Thank you for checking and reviewing.